Encapsulation Exercises/amulong-encapsulation-exercise-currency/src/com/amulong/encapsulation/exercise/currency/CurrencyConverter.java

```diff
@@ -0,0 +1,81 @@
+package com.amulong.encapsulation.exercise.currency;
+
+public class CurrencyConverter {
+    private double pesoToUsd;
```

Comment:
you do not need all of these. you can have a value field and currency
`private double value;
private String currency;`
then use these fields for conversion